code/game/gamemodes/miniantags/revenant/revenant_abilities.dm

```diff
@@ -250,7 +250,7 @@
 					human.adjustToxLoss(toxdamage)
 					human.AdjustConfused(confusion, bound_lower = 0, bound_upper = maxconfusion)
 					new/obj/effect/temp_visual/revenant(human.loc)
-				if(!istype(T, /turf/simulated/wall/rust) && !istype(T, /turf/simulated/wall/r_wall) && istype(T, /turf/simulated/wall) && prob(15))
+				if(!istype(T, /turf/simulated/wall/indestructible) && !istype(T, /turf/simulated/wall/rust) && !istype(T, /turf/simulated/wall/r_wall) && istype(T, /turf/simulated/wall) && prob(15))
```

Comment:
Don't really like how this code is just a massive `istype()` chain, that can be refactored separately. 👍